### ISSUE TITLE
Allow Site Settings to be resolved within URL

### ISSUE THREAD

**Issue by Deagan** [spikecodes/libreddit#410]
## Is your feature request related to a problem? Please describe.
Currently, site settings can only be set with a settings link that then saves it into a cookie. Site settings can be expanded to allow for a personalized settings URL, which can be used to set settings via URL alone.

## Describe the solution you'd like
I would like to apply my site settings by appending the settings directly to the URL. This would allow me to share links with specific settings, and apply Libreddit settings without using cookies.

## Describe alternatives you've considered
I do not store cookies on my devices across sessions. I attempted to create a userscript that would redirect to the Customer Settings URL before visiting the proper Libreddit page, but it seems this is not possible.

## Additional context
Search engines like Startpage and SearX generate a unique URL with the settings inside of it, and search queries are appended at the end of the URL to commit the actual search. I assume something similar can be done using the Libreddit Settings URL already implemented?

**Comment by Export33** [comment]
Yes, this feature would be great.

**Comment by EdwardLangdon** [comment]
@spikecodes Helpful for Tor Users

**Comment by Export33** [comment]
I delete my cookies each time i close my browser so having a link with the url in my favorites would help me a lot.

**Comment by BinaryGenius** [comment]
I use this url so I can save my eyes https://libreddit.strongthany.cc/ I just wish tor auto redirected reddit links to libreddit that would be an enormous time saver for me.

**Comment by Deagan** [comment]
Following similar Feature Requests for Nitter here:

https://github.com/zedeus/nitter/issues/553

**Comment by BinaryGenius** [comment]
Yeah I'm on the fence about using redirecting extensions on Tor the foundation is heavily against any modification of the browser.

**Comment by Export33** [comment]
+1 for url setting. Having a workaround is good but i don´t like the idea to install another plugin for that.
